interface/forms/newpatient/templates/newpatient/partials/common/fields/_issues.html.twig

```diff
@@ -0,0 +1,26 @@
+<div class="col-sm {{ displayOptionClass('enc_enable_issues') }}">
+    <fieldset>
+        <legend>{{ "Link/Add Issues to This Visit"|xlt }}</legend>
+        <div id="visit-issues">
+            <div class="form-row px-3">
+                {% if canAddIssues %}
+                    <div class="pb-1 col-sm">
+                        <a href="{{ webroot }}/interface/patient_file/summary/add_edit_issue.php"
+                           class="btn d-block btn-primary btn-add btn-sm enc_issue"
+                           onclick="top.restoreSession()">{{ "Add Issue"|xlt }}</a>
+                    </div>
+                {% endif %}
+                <select multiple name="issues[]" class="form-control">
+                    {% for issue in issues %}
+                        <option value="{{ issue.id|attr }}" {{ issue.selected ? 'selected' : '' }}>
+                            {{ issue.type|text }}: {{ issue.title|text }}
+                        </option>
+                    {% endfor %}
+                </select>
+                <p><i>{{ 'To link this encounter/consult to an existing issue, click the '
+                        ~ 'desired issue above to highlight it and then click [Save]. '
+                        ~ 'Hold down [Ctrl] button to select multiple issues.'|xlt }}</i></p>
```

Comment:
translation constants need to be kept on one line. I know it is ugly int he code, but needed for it to work.

Reply:
k, I was trying to translate from the previous file which was also doing string concatenation.  Turned it into one line.

Reply:
Fixed.

Reply:
that's pretty clever with twig's `~` but doesn't the translate thing scan the code in order to build a file of constants

Reply:
Yea, sadly, that's why it has to be on one line.